Tiltfile

```diff
@@ -0,0 +1,326 @@
+    yaml = str(kustomizesub("./config/default"))
+
+    # add extra_args if they are defined
+    if settings.get("extra_args"):
+        hivelocity_extra_args = settings.get("extra_args").get("hivelocity")
+        if hivelocity_extra_args:
+            yaml_dict = decode_yaml_stream(yaml)
+            append_arg_for_container_in_deployment(
+                yaml_dict, "caphv-controller-manager", "capi-hivelocity-system", "manager", hivelocity_extra_args)
+            yaml = str(encode_yaml_stream(yaml_dict))
+            yaml = fixup_yaml_empty_arrays(yaml)
+
+    # Set up a local_resource build of the provider's manager binary.
+    local_resource(
+        "manager",
+        cmd='mkdir -p .tiltbuild;CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags \'-extldflags "-static"\' -o .tiltbuild/manager',
+        deps=["api", "config", "controllers",
+              "pkg", "go.mod", "go.sum", "main.go"],
+        labels=["CAPHV"],
+    )
+
+    entrypoint = ["/manager"]
+    extra_args = settings.get("extra_args")
+    if extra_args:
+        entrypoint.extend(extra_args)
+
+    # Set up an image build for the provider. The live update configuration syncs the output from the local_resource
+    # build into the container.
+    docker_build_with_restart(
+        ref="ghcr.io/hivelocity/caphv-staging",
+        context="./.tiltbuild/",
+        dockerfile_contents=tilt_dockerfile_header,
+        target="tilt",
+        entrypoint=entrypoint,
+        only="manager",
+        live_update=[
+            sync(".tiltbuild/manager", "/manager"),
+        ],
+        ignore=["templates"],
+    )
+
+    k8s_yaml(blob(yaml))
+    k8s_resource(workload="caphv-controller-manager", labels=["CAPHV"])
+    k8s_resource(
+        objects=[
+            "capi-hivelocity-system:namespace",
+            "hivelocitymachines.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+            "hivelocitymachinetemplates.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+            "hivelocityclusters.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+            "hivelocityclustertemplates.infrastructure.cluster.x-k8s.io:customresourcedefinition",
+            "caphv-mutating-webhook-configuration:mutatingwebhookconfiguration",
+            "caphv-controller-manager:serviceaccount",
+            "caphv-leader-election-role:role",
+            "caphv-manager-role:clusterrole",
+            "caphv-metrics-reader:clusterrole",
+            "caphv-leader-election-rolebinding:rolebinding",
+            "caphv-manager-rolebinding:clusterrolebinding",
+          #  "caphv-manager-config:configmap", # question: not found
+            "caphv-serving-cert:certificate",
+            "caphv-selfsigned-issuer:issuer",
+            "caphv-validating-webhook-configuration:validatingwebhookconfiguration",
+        ],
+        new_name="caphv-misc",
+        labels=["CAPHV"],
+    )
+
+    if os.path.exists("baremetalhosts.yaml"):
```

Comment:
I think we can remove this part

Reply:
thank you for the hint, this is already removed.